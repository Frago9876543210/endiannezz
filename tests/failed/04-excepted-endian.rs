use endiannezz::Io;

#[derive(Io)]
#[endian()]
struct Foo;

fn main() {}
