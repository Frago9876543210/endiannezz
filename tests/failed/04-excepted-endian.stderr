error: excepted endian
 --> $DIR/04-excepted-endian.rs:4:1
  |
4 | #[endian()]
  | ^^^^^^^^^^^
